README.md

```
@@ -0,0 +1,84 @@
+```bash
+python cloc.py --path . --start-date 2024-01-01 --end-date 2024-12-31
+```
+
+### Xuất báo cáo ra file
+```bash
+python cloc.py --path . --output-format json --output-file report.json
+```
+
+## Cấu trúc dự án
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)